subprojects/packagefiles/aklomp-base64/meson.build

```
@@ -0,0 +1,193 @@
+    }
+  elif is_arm
+    arch_flags += {
+      'NEON32': [],
+    }
+  endif
+else
+  if is_x86
+    arch_flags += {
+      'SSSE3': ['-mssse3'],
+      'SSE41': ['-msse4.1'],
+      'SSE42': ['-msse4.2'],
+      'AVX': ['-mavx'],
+      'AVX2': ['-mavx2'],
+      'AVX512': ['-mavx512vl', '-mavx512vbmi'],
+    }
+  elif is_arm
+    arch_flags += {
+      'NEON32': ['-mfpu=neon'],
+    }
+  endif
+endif
+
+foreach feature, flags : arch_flags
+  present = cc.has_multi_arguments(flags)
+  config.set10('BASE64_WITH_' + feature, present)
+  if not present
+    arch_flags += {
+      feature: [],
+    }
+  endif
+endforeach
+
+cfg = configure_file(
+  input: 'cmake/config.h.in',
+  output: 'config.h',
+  configuration: config,
+  format: 'cmake',
+)
+
+codecs = []
+
+codecs += [
+  static_library(
+    'aklomp-base64-generic',
+    'lib/arch/generic/codec.c',
+    'lib/tables/tables.c',
+    cfg,
+    c_args: c_args,
+    override_options: ['c_std=c99'],
+  ),
+  static_library(
+    'aklomp-base64-ssse3',
+    'lib/arch/ssse3/codec.c',
+    cfg,
+    c_args: c_args + arch_flags.get('SSSE3', []),
+    override_options: ['c_std=c99'],
+  ),
+  static_library(
+    'aklomp-base64-sse41',
+    'lib/arch/sse41/codec.c',
+    cfg,
+    c_args: c_args + arch_flags.get('SSE41', []),
+    override_options: ['c_std=c99'],
+  ),
+  static_library(
+    'aklomp-base64-sse42',
+    'lib/arch/sse42/codec.c',
+    cfg,
+    c_args: c_args + arch_flags.get('SSE42', []),
+    override_options: ['c_std=c99'],
+  ),
+  static_library(
+    'aklomp-base64-avx',
+    'lib/arch/avx/codec.c',
+    cfg,
+    c_args: c_args + arch_flags.get('AVX', []),
+    override_options: ['c_std=c99'],
+  ),
+  static_library(
+    'aklomp-base64-avx2',
+    'lib/arch/avx2/codec.c',
+    cfg,
+    c_args: c_args + arch_flags.get('AVX2', []),
+    override_options: ['c_std=c99'],
+  ),
+  static_library(
+    'aklomp-base64-avx512',
+    'lib/arch/avx512/codec.c',
+    cfg,
+    c_args: c_args + arch_flags.get('AVX512', []),
+    override_options: ['c_std=c99'],
+  ),
+  static_library(
+    'aklomp-base64-neon32',
+    'lib/arch/neon32/codec.c',
+    cfg,
+    c_args: c_args + arch_flags.get('NEON32', []),
+    override_options: ['c_std=c99'],
+  ),
+  static_library(
+    'aklomp-base64-neon64',
+    'lib/arch/neon64/codec.c',
+    cfg,
+    c_args: c_args,
+    override_options: ['c_std=c99'],
+  ),
+]
+
+src = [cfg, 'lib/lib.c', 'lib/codec_choose.c']
+
+inc = include_directories('include')
+
+libbase64 = library(
+  'base64',
+  src,
+  c_args: c_args,
+  include_directories: inc,
+  link_with: codecs,
+  install: true,
+  version: meson.project_version(),
+  override_options: ['c_std=c99'],
+  gnu_symbol_visibility: 'hidden',
+)
+
+base64_dep = declare_dependency(
+  include_directories: inc,
+  link_with: libbase64,
+)
+
+install_headers('include/libbase64.h')
+
+meson.override_dependency('base64', base64_dep)
+
+pkg.generate(
+  libbase64,
+  name: 'aklomp-base64',
```

Must match the provide name.

```suggestion
  name: 'base64',
```